pkg/kubelet/im/image_gc.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
Disk manager should abstract out Image GC from the rest of kubelet. We are not there yet though.